javaslang-gwt-example/README.md

```diff
@@ -0,0 +1,17 @@
+# GWT support for Javaslang
+
+### Running the example
+
+* Install all necessary artifacts by running the following command in the main project's directory:
```

Comment:
I tried many times to avoid this step, but unsuccessful. Also I've never seen other solution for maven projects. When (and if) we will migrate to gradle all things will be simpler